index.html

```diff
@@ -68,6 +64,216 @@ <h2 class="connect">LET’S CONNECT</h2>
+              or sign-ups required.
+            </p>
+            <ul class="portfolio__tags">
+              <li class="tag">HTML</li>
+              <li class="tag">CSS</li>
+              <li class="tag">JavaScript</li>
+            </ul>
+            <div class="button_primary">
+              <a>See project</a>
+            </div>
+          </div>
+        </article>
+      </div>
+    </section>
+    <section
+				class="section section__white section__white--top"
+				id="about_me"
+			>
+				<div class="about">
+					<h2 class="about_title">About Myself</h2>
+					<p class="color-info">
+						Hello I'm a software developer! I can help you build a product,
+						feature or website. Look through some of my work and experience!
+						If you like what you see and have a project you need coded,
+						don't hesitate to contact me.
+					</p>
+					<h2 class="connect">LET’S CONNECT</h2>
+					<ul class="social_icons flex-row align-center">
+						<li><a href=""><img src="assets/twitter.png" alt="Twitter" /></a></li>
+						<li><a href=""><img src="assets/Linkedin.png" alt="LinkedIn" /></a></li>
+						<li><a href=""><img src="assets/medium.png" alt="Medium" /></a></li>
+						<li><a href=""><img src="assets/github.png" alt="Github" /></a></li>
+						<li><a href=""><img src="assets/angel.png" alt="Angel" /></a></li>
+					
+					</ul>
+					</div>
+					<div>
+						<a  class="button_primary">Get my resume</a>
+					</div>
+					<div class="category-about_page">
+						<header
+							class="about_page__header flex-row justify-between align-center"
+						>
+							<h3 class="about_page__subtitle">Languages</h3>
+							<img src="assets/arr_d.svg" alt="Arrow down icon" />
+						</header>
+						<ul class="about_page__list grid">
+							<li class="about_page__item">
+								<img
+									src="assets/Ellipse1.png"
+									alt="javascript png image"
+								/>
+								JavaScript
+							</li>
+							<li class="about_page__item">
+								<img
+									src="assets/Ellipse2.png"
+									alt="html png image"
+								/>
+								HTML
+							</li>
+							<li class="about_page__item">
+								<img
+									src="assets/Ellipse3.png"
+									alt="css png image"
+								/>
+								CSS
+							</li>
+						</ul>
+					</div>
+					<div class="category-about_page">
+						<header
+							class="about_page__header flex-row justify-between align-center"
+						>
+							<h3 class="about_page__subtitle">Frameworks</h3>
+							<img
+								src="assets/arr_r.svg"
+								alt="Arrow right icon"
+							/>
+						</header>
+					</div>
+					<hr />
+					<div class="category-about_page">
+						<header
+							class="about_page__header flex-row justify-between align-center"
+						>
+							<h3 class="about_page__subtitle">Skills</h3>
+							<img
+								src="assets/arr_r.svg"
+								alt="Arrow right icon"
+							/>
+						</header>
+					</div>
+					<hr />
+				</div>
```

Comment:
- [ ] Nice work Implementing the portfolio cards and layout, upon careful examination I notice your design does not correspond with the Figma design, some of the key things you need to implement
 - [ ] flex
 - [ ] margin
 - [ ] padding 

- [ ] Your Design
![image](https://github.com/montsecruz9/personal-portfolio/assets/68823331/8153d03b-6708-4766-8376-d239f10dd2e5)


- [ ] Figma Design
![image](https://github.com/montsecruz9/personal-portfolio/assets/68823331/d781ef0b-0d1b-4c31-b80a-bd4853933209)

Reply:
Hi @Cabraham1 ! Thanks for your comments :) Do you have any suggestions on how to put those arrows the way they are like in the figma design?